app/views/games/show.html.erb

```diff
@@ -1,9 +1,24 @@
 <h1 class="text-center">Have Fun</h1>
 
+<!-- drawing chess board -->
 <table class='chess-board chess-squares'>
-  <% @rows.times do |row| %>
+  <% 8.times do |row| %>
     <tr>
-      <% @columns.times do |column| %><td id="<%= (row + 1) * (column + 1) %>"></td><% end %>
+      <% 8.times do |column| %><td id="<%= (row * 8) + (column + 1) %>"></td><% end %>
```

Comment:
See the note below first....

Could we do this like this instead (I'm sure this code is super buggy, but it gets the point across):

```
<% 8.times do |column| %>
  <td id="<%= (row * 8) + (column + 1) %>">
      <% if @game.pieces.where(:x_coord => row, :y_coord => column) %>
           GLYPH-ICON STUFF HERE
     <% end %>
  </td>

<% end %>


```